sourcetable.go

```diff
@@ -123,3 +127,163 @@ func (m MountEntry) String() string {
 		m.CountryCode, m.Latitude, m.Longitude, nmea, solution, m.Generator, m.Compression,
 		m.Authentication, fee, m.Bitrate, m.Misc)
 }
+
+// DecodeSourcetable parses a sourcetable from an ioreader into a ntrip style source table.
+func DecodeSourcetable(in []byte) (Sourcetable, []error) {
```

Comment:
I think it makes most sense for this to take a string rather than a byte slice (given it's effectively the inverse of the Sourcetable.String() method.